types/jasmine/jasmine-tests.ts

```diff
@@ -1443,6 +1443,15 @@ describe("createSpyObj", function() {
         expect(spyObj.method2.and.identity()).toEqual('BaseName.method2');
     });
 
+    it("should allow you to provide properties with values to jasmine.createSpyObj()", function() {
```

Comment:
Notice, this code is never executed and it's sole purpose it just to describe types.. Therefore, it would be more clear to write scenarios from this perspective, as it allows to omit some parts (e.g. assertion part is not necessary). Please cover all 4 creation methods you have just introduced. Also ensure to have test cases with non-existing property names, so it errors in that line. For both, array and object syntax.